openstack/lyra/templates/runs-pruning.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+
+metadata:
+  name: runs-pruning
```

Comment:
lets call this something more generic: db-pruner (also rename the file)

Reply:
ok